### PR TITLE
Noetic Dockerfile minor fixes

### PR DESCRIPTION
**Pull request by mabelzhang**
Made a few minor changes to the Noetic Dockerfile while I was testing tutorials:

- Fix formatting, alphabetical order
- Add Vim, because I need Vim to do anything
- Gazebo installation key no longer hardcoded. It `wget`s a `gazebo.key` file, as in the [Gazebo installation tutorials](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install).

### CODE REVIEW

**Comment by mabelzhang** [comment]
The static code check is failing on files I didn't touch :sweat_smile: I think that means they are in the reorg branch and will need to be fixed in another PR before we can merge the reorg branch into `master`.

**Comment by dtdavi1** [comment]
I'm having trouble getting the Docker version running correctly, so I'm not a good choice to review this one (I'll work with someone offline once the release dust settles to figure out what I'm doing wrong).

**Comment by bsb808** [comment]
I am looking at this, but am not able to get catkin_make to run successfully from the docker container.  @mabelzhang - is it working for you?

**Comment by mabelzhang** [comment]
It works for me. What's the error you're getting?
Not sure if it makes a difference, but I'm using the code from #120 (I had to make that fix during the build), running in a container from this Dockerfile.

**Comment by bsb808** [comment]
I was able to sort it out.  The problem was that the vcs repository list was still pointing to the original uuv_simulator repo - not our fork, which we updated for Noetic.

I made some additions to your branch in #121

**Review by bsb808 — APPROVED** [review]
Tested with code in #121 on top of this and works fine.

**Comment by bsb808** [comment]
@mabelzhang I made a mess out of this.  I expect that @M1chaelM 's work on a general docker image will alleviate the need for this PR.  Let's put this on hold for now.

**Comment by M1chaelM** [comment]
I think we can close this now. The 3 original changes have been added to images in the [dockwater repo](https://github.com/Field-Robotics-Lab/dockwater).

**Comment by bsb808** [comment]
I agree that we can close, but I think it is most consistent with our workflow to have the author close as method of concurrence.

**Pull request closed without merging**
